lisa/tests/scheduler/load_tracking.py

```diff
@@ -309,9 +309,18 @@ def _test_task_signal(self, signal_name, allowed_error_pct,
 
         signal_mean = area_under_curve(signal) / (signal.index[-1] - signal.index[0])
 
-        if signal_name == 'load':
-            # Load isn't CPU invariant
-            exp_signal /= (self.plat_info['cpu-capacities'][cpu] / UTIL_SCALE)
+        # Since load is now CPU invariant in recent kernel versions, we don't
+        # rescale it back. To match the old behavior, that line is
+        # needed:
+        #  exp_signal /= (self.plat_info['cpu-capacities'][cpu] / UTIL_SCALE)
+        kernel_version = self.plat_info['kernel']['version']
+        if (
+            signal_name == 'load'
+            and kernel_version.parts < (5, 1, 0)
```

Comment:
Just wondering, how does that work with rc versions ? Is 5.1-rc1 < 5.1.0 ?

Reply:
Good point, "rc" is not in `.parts` tuple, but will lead to `minor=None`, so the comparison will raise an exception if we get to that. Instead, I can replace it with `kernel_version.parts[:2] < (5, 1)` to just ignore the rc and minor numbers (comparing tuples with different number of parts works but seems to give surprising results wrt equality).